### DISCUSSION TITLE
NexoPOS v4.8.14

### DISCUSSION THREAD

**Discussion by nexopos** [Blair2004/NexoPOS#1304] [Releases]
This version comes with a bunch of bug fixes that was recently reported by our community. While NexoPOS 5x remains on development status, we'll make sure to merge those change there to ease furthermore the upgrade. Here are the list of changes that was made during this update:

You're invited from now not to use **My NexoPOS** to update module. In fact, we noticed that the way the app was updated is not completely safe and keeps previous release files which is not appropriate. While we're working on an upgrade for NexoPOS 5, you're invited to update manually. Please let us know via contact@nexopos.com if you have any issue (Premium Support).

Changelog:
=======
- Fix: double entry on cash flow for procurement that has status updated from list (https://github.com/Blair2004/NexoPOS/issues/1265)
- Added: New language German
- Fixed: flat discount on product was ignored
- Updated: improving code readability
- Added: new default units for demo
- Updated: demo know creates grouped product
- Updated: we've simplified the computation of unit value on grouped products
- Updated: searching products on procurement excludes grouped products
- Added: new columns on products history to track order products id
- Updating Dependencies

We would like to give a special thanks to @swurzinger for his contribution on the localization.

<hr /><em>This discussion was created from the release <a href='https://github.com/Blair2004/NexoPOS/releases/tag/v4.8.14'>NexoPOS v4.8.14</a>.</em>

**Comment by killuadk** [comment]
How to update manually for those who have already DB filled
